frontend-react/src/components/MessageTracker/MessageReceivers.tsx

```diff
@@ -329,32 +334,20 @@ export const MessageReceivers = ({ receiverDetails }: MessageReceiverProps) => {
         // which requires a return of -1 0 1. For dates, we need to
         // convert them to a numerical value.
         () =>
-            activeColumnSortOrder !== FilterOptionsEnum.NONE
+            activeColumnSortOrder !== FilterOptionsEnum.NONE && activeColumn
```

Comment:
Out of the scope of this pull request, but I think it'd be nice to pull out the normalization logic and the sorting logic out of the render cycle and into more unit-testable functions.  I think that'd make it easier to write a regression test for this bug too 🤔

Reply:
too bad, you get it meow